agenda-services/src/main/java/org/exoplatform/agenda/util/EntityMapper.java

```diff
@@ -477,6 +477,19 @@ public static EventAttendeeEntity toEntity(EventAttendee eventAttendee) {
     return eventAttendeeEntity;
   }
 
+  public static GuestUserEntity toEntity(GuestUser guestUser) {
+    GuestUserEntity guestUserEntity = new GuestUserEntity();
+    guestUserEntity.setId(guestUser.getId());
+    guestUserEntity.setGuestEmail(guestUser.getGuestEmail());
```

Comment:
what about the eventId ?

Reply:
> what about the eventId ?

guestUserEntity has as attribute the event object not eventId  so I put it setEvent().